src/components/TasksTable/index.jsx

```diff
@@ -33,6 +35,10 @@ const sorted = pipe(
   },
 }))
 export default class TasksTable extends Component {
+  state = {
+    showFilterContent: false,
```

Comment:
Will assignee be the only filter option we show? If this is the case, then maybe when a user clicks on the filter icon, we display the `<TextField select ... />` directly. If there's only one filter planned (at least for the MVP) then we don't need to have the dedicated filter content area.

To show which filter is selected, I can see us putting a check (✓) icon beside the menu item name.

Reply:
In that case, maybe having the dedicated filter content area makes sense.